docs/INSTALL.md

```diff
@@ -192,9 +192,17 @@ dictionary:
   reportWorkflow:
     nbOfHoursQuarantineApproved: 24
 
-  maxTextFieldLength: 250
+  # Add exceptions if there is any, only `default` is mandatory, others are optional
+  maxTextFieldLength:
+    default: 250
+    report:
+      nextSteps: 1000
+      intent: 400
+      keyOutcomes: 600
+    authorizationGroup:
+      description: 400
 
-  fields:
+fields:
```

Comment:
If you want to remove the spaces here, remove the top-level `dictionary:` key (as that no longer exists) and shift everything left by two spaces (docs are incorrect currently).